yarn.lock

```diff
@@ -3052,7 +3052,7 @@
     fp-ts "^2.12.1"
     io-ts "^2.2.16"
 
-"@pagopa/openapi-codegen-ts@^12.0.1":
+"@pagopa/openapi-codegen-ts@^12.0.2":
```

Comment:
Strange change because the package.json is not changed and the resolved version as written in the  line below is already 12.0.2.
Why you think this occurred?

Reply:
What does it mean that the package.json file hasn't changed? In this branch the `package.json` has been updated with right packages version above.